pkg/tfgen/docs.go

```diff
@@ -1169,17 +1170,25 @@ func (g *Generator) convertHCL(hcl, path string) (string, string, error) {
 		err = convertHCL("go")
 	case Schema:
 		langs := []string{"typescript", "python", "csharp", "go"}
+		var anySucceeded bool = false
 		for _, lang := range langs {
 			if langErr := convertHCL(lang); langErr != nil {
-				return "", stderr.String(), langErr
+				err = multierror.Append(err, langErr)
```

Comment:
It would be nice if we could still capture stderr here. If you want to take care of that with a separate PR, please file an issue for followup.

Reply:
I am working on expanding coverage and diagnostics tracking, and will make sure that stderr is captured even if the language isn't able to convert.